### PR TITLE
ODC Known Issues and CRC entry in RN

### PR DESCRIPTION
**Pull request by Preeticp**
* Added notes on RBAC and serverless app delays
* Adds KIs in the KI section in the release notes for odc
* Adds entry for CRC in the the 4.2 RN
All these have been Acked by QE and SMEs
This is meant for the 4.2 version

### CODE REVIEW

**Comment by Preeticp** [comment]
@ahardin-rh request you to review this PR.

File: applications/application-life-cycle-management/odc-creating-applications-using-developer-perspective.adoc [hunk 1]
```diff
@@ -26,5 +26,6 @@ To create applications using the *Developer* perspective ensure that:
 
 * You have xref:../../web-console/web-console.adoc#web-console[logged in to the web console].
 * You are in the xref:../../web-console/odc-about-developer-perspective.adoc#odc-about-developer-perspective[*Developer* perspective].
+* You have the appropriate link:https://docs.openshift.com/container-platform/latest/authentication/using-rbac.html#default-roles_using-rbac[roles and permissions] in a project to create applications and other workloads in {product-title}.
```
Comment (ahardin-rh) [inline applications/application-life-cycle-management/odc-creating-applications-using-developer-perspective.adoc]: For internal links, we must follow this format: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#internal-cross-references

So, the link should look something like:

xref:../authentication/using-rbac.adoc#default-roles_using-rbac[roles and permissions]

File: applications/application-life-cycle-management/odc-viewing-application-composition-using-topology-view.adoc [hunk 2]
```diff
@@ -11,6 +11,7 @@ To view your applications in the *Topology* view and interact with them, ensure
 
 * You have xref:../../web-console/web-console.adoc#web-console[logged in to the web console].
 * You are in the xref:../../web-console/odc-about-developer-perspective.adoc#odc-about-developer-perspective[*Developer* perspective].
+* You have the appropriate link:https://docs.openshift.com/container-platform/latest/authentication/using-rbac.html#default-roles_using-rbac[roles and permissions] in a project to create applications and other workloads in {product-title}.
```
Comment (ahardin-rh) [inline applications/application-life-cycle-management/odc-viewing-application-composition-using-topology-view.adoc]: Same for this link. Update to this format:

xref:../authentication/using-rbac.adoc#default-roles_using-rbac[roles and permissions]

File: modules/odc-viewing-application-topology.adoc [hunk 3]
```diff
@@ -5,9 +5,14 @@
...
 
-A serverless application is visually indicated with the Knative symbol (image:odc_serverless_app.png[title="Serverless Application"]). You can zoom in and out to see more details for a particular application.
+A serverless application is visually indicated with the Knative symbol (image:odc_serverless_app.png[title="Serverless Application"]).
+
+[NOTE]
+====
+Serverless applications take some time to load and display on the *Topology* view. When you create a serverless application, it first creates a service resource and then a revision, after which it is deployed and displayed on the Topology view. If it is the only workload, you may be redirected to the *Add* page. Once the revision is deployed, the serverless application is displayed on the *Topology* view.
```
Comment (ahardin-rh) [inline modules/odc-viewing-application-topology.adoc]: service resource, and then a revision. After that, it is deployed and displayed in the *Topology* view.

s/may be/might be

File: release_notes/ocp-4-2-release-notes.adoc [hunk 4]
```diff
@@ -278,6 +278,11 @@ within the {product-title} cluster for deployment. It detects changes to local
 code and deploys it to the cluster automatically, giving instant feedback to
 validate changes in real time. It supports multiple languages and frameworks.
 
+[id="ocp-4-2-CRC"]
+==== CodeReady Containers
+
+CodeReady Containers provides a local desktop instance of a minimal {product-title} 4 or newer cluster. This cluster provides developers with a minimal environment for development and testing purposes. It includes the `crc` CLI to interact with the CodeReady Containers virtual machine running the OpenShift cluster.
```
Comment (ahardin-rh) [inline release_notes/ocp-4-2-release-notes.adoc]: s/provides/provide

File: release_notes/ocp-4-2-release-notes.adoc [hunk 5]
```diff
@@ -1431,3 +1436,13 @@ It is recommended to delete the default StorageClass and allow the
 ClusterStorageOperator to recreate this with the correct attributes after
 the upgrade is completed.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1751641[*BZ#1751641*])
+
+* Non-serverless workloads show resources for serverless workloads in the Topology resources panel. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1760810[*BZ#1760810*])
+
+* Pod status is depicted inconsistently in the Topology view, resources side panel, and the deployment configs details page. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1760827[*BZ#1760827*])
+
+* Deploy image ignores the target port selection in the create application workflow and always uses the first entry. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1760836[*BZ#1760836*])
```
Comment (ahardin-rh) [inline release_notes/ocp-4-2-release-notes.adoc]: s/Deploy image/The deploy image function ignores

File: release_notes/ocp-4-2-release-notes.adoc [hunk 6]
```diff
@@ -1431,3 +1436,13 @@ It is recommended to delete the default StorageClass and allow the
...
+
+* Certain features like the name of the application and the build status are not rendered in the Topology view on the Edge browser. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1760858[*BZ#1760858*])
+
+* Determination of active pods when a rollout fails can be incorrect in the Topology view. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1760828[*BZ#1760828*])
```
Comment (ahardin-rh) [inline release_notes/ocp-4-2-release-notes.adoc]: I think we have **Topology** view elsewhere, so we should keep consistent if we're going to embolden it.

File: release_notes/ocp-4-2-release-notes.adoc (hunk 4 above)
Comment (robin-owen) [reply release_notes/ocp-4-2-release-notes.adoc]: It's actually a singular -- "CodeReady Containers" is the name of one product/project. :)

**Comment by ahardin-rh** [comment]
@Preeticp Just a few minor comments from me. Once those changes are made and they're squashed into one commit, we should be good to go! Thanks!

File: release_notes/ocp-4-2-release-notes.adoc (hunk 4 above)
Comment (Preeticp) [reply release_notes/ocp-4-2-release-notes.adoc]: Yes Kevin, retained provides.

**Comment by Preeticp** [comment]
@ahardin-rh thank you so much. Fixed the comments and squashed the commits.

**Comment by ahardin-rh** [comment]
Thanks, Preeti! Looks great.

**Pull request merged**